Makefile

```diff
@@ -32,7 +32,7 @@ DATAMGR_BIN ?= data-manager-for-plugin
 BACKUPDRIVER_BIN ?= backup-driver
 
 RELEASE_REGISTRY = vsphereveleroplugin
-REGISTRY ?= dpcpinternal
+REGISTRY ?= vsp-docker-local.artifactory.eng.vmware.com
```

Comment:
This should be a separate commit, not part of this PR.

Reply:
It might not be a good idea to put an internal registry as the default in an open source project.

Reply:
Good point.